keyboards/keychron/q2/mcuconf.h

```diff
@@ -16,12 +16,12 @@
 
 /*
  * This file was auto-generated by:
- *    `qmk chibios-confmigrate -i keyboards/acheron/arctic/mcuconf.h -r platforms/chibios/GENERIC_STM32_F072XB/configs/mcuconf.h`
+ *    `qmk chibios-confmigrate -i keyboards/keychron/q2/mcuconf.h -r platforms/chibios/boards/QMK_PROTON_C/configs/mcuconf.h`
```

Comment:
Just delete this whole comment block. It's injected when it's been migrated, but serves no purpose when it's a new board. No need to keep it.